bin/laminas-migration

```diff
@@ -23,9 +22,7 @@ if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     exit(1);
 }
 
-$version = strstr(Versions::getVersion('laminas/laminas-migration'), '@', true);
-
-$application = new Application('laminas-migration', $version);
+$application = new Application('laminas-migration', PackageVersions::getAppVersion());
```

Comment:
This seems to be the only use case for `PackageVersions`, therefore i stripped it down to this one.